text/0000-add-author.md

```diff
@@ -0,0 +1,60 @@
+# Meta
+[meta]: #meta
+- Name: Add Author to RFC Metadata
+- Start Date: 2020-03-14
+- CNB Pull Request: https://github.com/buildpacks/rfcs/pull/64
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This is a proposal to add an `Author(s):` field to the `Meta` section of the RFC template.
+
+# Motivation
+[motivation]: #motivation
+
+When an RFC is merged, and then renamed, the commit history is lost. This makes it difficult to remember who wrote the RFC. Sometimes, an RFC is written by more than one person, but the linked PR will not always show this.
+
+# What it is
+[what-it-is]: #what-it-is
+
+A new `Author(s):` field in the Meta section of the RFC template.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+We will add the following to the RFC `Meta` section:
+
+```
+- Author(s): (Github usernames)
+- RFC Pull Request: (leave blank)
+```
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Even though the author will be know, the commit history is still hard to find.
+
+# Alternatives
+[alternatives]: #alternatives
+
+## RFC PR Link
+
+Instead of an author field, we could only enforce that a link to the original RFC PR be added to the RFC after it's merged. This would make it easier to find the author and the history, but we would need to ensure that all authors are represented in the PR.
+
+# Prior Art
+[prior-art]: #prior-art
+
+- Rust uses an ["RFC PR" link](https://github.com/rust-lang/rfcs/blob/master/0000-template.md)
+- TensorFlow has an ["Author(s)" field](https://github.com/tensorflow/community/blob/master/rfcs/yyyymmdd-rfc-template.md)
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+- Does this make it look like only a few person contributed to the RFC (when in reality they are a collaborative process)?
```

Comment:
Maybe, but with the easy access to the discussion you can always check the history. We could also encourage / welcome more folks to the RFC process. :)